flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraph.java

```diff
@@ -85,6 +86,7 @@
 	private Set<Integer> sources;
 	private Set<Integer> sinks;
 	private Map<Integer, Tuple2<Integer, List<String>>> virtualSelectNodes;
+	private Map<Integer, Tuple2<Integer, OutputTag>> virtualOutputNodes;
```

Comment:
The method is already called `addVirtualSideOutputNode()`. I'm adjusting the name of the field. Thanks!